Packs/ThycoticDSV/pack_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "ThycoticDSV",
+    "description": "Thycotic\u2019s DevOps Secrets Vault is a high velocity vault that centralizes secrets management, enforces access, and provides automated logging trails. This cloud-based solution is platform agnostic and designed to replace hard-coded credentials in applications, micro-services, DevOps tools, and robotic process automation. This vault ensures IT, DevOps and Security teams the speed and agility needed to stay competitive without sacrificing security.",
```

Comment:
This description also appears in the README and the integration's description.
Please visit our documentation website or look at existing integrations to see the differences